### DISCUSSION TITLE
GM Only Able to Use Mod, Players Only Send in Common

### DISCUSSION THREAD

**Discussion by arizonagarbage** [mclemente/fvtt-module-polyglot#204] [Q&A] [answered]
I saw a thread already declared "Answered" but I am having the exact same issue. I have uninstalled and reinstalled the module, rebuilt my characters' sheets, and reset everything in the module back to their defaults. But only the GM account can use the module.

The module worked previously, but now only I as the GM can send things in other languages. If my players "understand" the language, they see it as the module intended. If they don't, they see the foreign language, but not the translation. As stated, everything works as intended when I, the GM, use it.

When my players attempt to use the module, even if they choose a different language, the upper foreign language and lower translations don't appear. The chat message looks like all other messages, as if they left the options on Common, even if they chose a different language. All other characters see the Common message even if they don't understand the language. They each control their own characters with assigned languages on the player sheet:

![image](https://user-images.githubusercontent.com/94332181/141709572-861b99e7-fb15-4c46-a152-9ea64e298ce1.png)

![image](https://user-images.githubusercontent.com/94332181/141709630-dff0953c-f94d-4008-a9a7-cd750eca40e5.png)

**Comment by mclemente** [comment, marked as answer]
Hey, could you zip your world and send it over to me?

**Reply by mclemente** [reply]
If a player doesn't have a character on the map, it is speaking OOC. Set your `Scramble OOC Chat Messages` for All.

**Reply by arizonagarbage** [reply]
Ahhhhh. Thank you!
